src/makeInfinityAdder.js

```diff
@@ -1,12 +1,35 @@
 'use strict';
 
 /***
- *
  * @return {function}
  */
-
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(value) {
+    if (arguments.length === 0) {
+      return sum;
```

Comment:
The current implementation does not reset the sum to 0 after returning it. This means subsequent calls to adder() without arguments will not return 0 as expected.